config/game_options.txt

```diff
@@ -597,3 +597,6 @@ MODETIER_VOTING
 
 ## Number of modes dropped by the modetier vote during mode selection, after vote.
 DROPPED_MODES 3
+
+## Whether the suicide verb is allowed.
+# SUICIDE_ALLOWED
```

Comment:
Fun police hates gloomy sunday rounds.

Reply:
i know what a "gloomy sunday" is but not a "gloomy sunday round"

Reply:
Sad.